src/Management/test/KubernetesCore.Test/HostBuilderExtensionsTest.cs

```diff
@@ -21,6 +21,7 @@ namespace Steeltoe.Management.Kubernetes.Test;
 public class HostBuilderExtensionsTest
 {
     [Fact]
+    [Trait("Category", "SkipOnLinux")]
```

Comment:
Are you sure skipping is the best course of action here? Have you confirmed that your changes are not related to these failing?

Reply:
No that was for debugging. Now the bug is fixed and the skipping removed.